src/components/shared/TreeView.js

```diff
@@ -258,6 +258,8 @@ class TreeViewRowScrolledColumns<
         aria-selected={selected}
         aria-label={ariaLabel}
         role="treeitem"
+        tabIndex={-1}
+        id={index}
```

Comment:
An ID should be unique across the document, so it's better if you use a prefix here. (and anyway it's advised that an id starts with a letter for legacy reasons).

Moreover it's very likely we want `nodeId` instead of `index`... because `index` is just the index in the currently rendered elements (this comes from: https://github.com/firefox-devtools/profiler/blob/2fb0fcb53f4eb30433e5d57a08fe1ac1d1c4be2b/src/components/shared/VirtualList.js#L61-L72)

I believe this is why this doesn't work like you expect it too :)

Reply:
Thanks @julienw !